### ISSUE TITLE
lib0 no longer supports nodejs 12

### ISSUE THREAD

**Issue by e-carlin** [radiasoft/download#321]
In fedora32 the default nodejs is v12.22.1. But lib0 [dropped support for it](https://github.com/dmonad/lib0/commit/deea8b173395b216af8f52482d7c063a7600847f). lib0 is a nested dependency of jupyter_rs_vtk (at least, probably all jupyter widgets depend on it). 

I think we can specify nodejs:16/default

**Comment by e-carlin** [comment]
@robnagler I've tried a couple different solutions but no success and I'd like your thoughts. 

Here is what I've tried:
1. specify `@nodejs:16/default` as the package to install. This works on the command line `dnf install -y @nodejs:16/default` but rpmbuild blows up because dependency names must start with `-\w'`. 
2. Run `dnf module enable -y nodejs:16/default` in common.sh and leave the package name to install as nodejs. This doesn't work because it doesn't seem there is any way of persisting that the module was enabled when `rpmbuild` is called. So, it just installs the default nodejsv12


Before I go any further I'm wondering if you have thoughts on how to resolve?

If needed I could make a nodejs rpm-code and install it through that.
